hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/RequestFactoryImpl.java

```diff
@@ -124,6 +124,11 @@ public class RequestFactoryImpl implements RequestFactory {
    */
   private final StorageClass storageClass;
 
+  /**
+   * Is Multipart Enabled
```

Comment:
. in the end for javadoc.

Reply:
m small, is multipart upload enabled. As this is only for uploads not downloads as well.

Reply:
Other config names clearly mentions it is multipart upload only. Why are we not using isMultipartUploadEnabled here as well?